tests/test_slack_client.py

```
@@ -994,3 +994,22 @@ def test_yaml_with_non_string_values_ignored(self, mock_webclient, mock_get_secr
+    def test_uses_shared_lambda_environment_helper(self, mock_env_info, mock_webclient, mock_get_secret, mock_boto3):
+        """Verify SlackClient delegates to get_lambda_environment_info"""
+        mock_get_secret.return_value = {"bot_token": "xoxb-test-token"}
+        mock_env_info.return_value = {
+            "function_name": "test-fn",
+            "function_version": "$LATEST",
+            "aws_region": "ap-southeast-2",
+            "environment": "unknown",
+            "memory_limit": "",
+            "is_local": True,
+        }
+        SlackClient(secret_name="test-secret")
```

Ruff (0.14.14)

[error] Possible hardcoded password assigned to argument: "secret_name"

(S106)